src/Symfony/Component/HttpFoundation/Tests/JsonResponseTest.php

```diff
@@ -280,12 +277,10 @@ public function testConstructorWithObjectWithoutToStringMethodThrowsAnException(
     }
 }
 
-if (interface_exists(\JsonSerializable::class, false)) {
```

Comment:
`JsonSerializable` has been introduced with PHP 5.4. I'm pretty confident it is always defined. 🙂